api-reference/50 Common/utils/zz Errors and Warnings/W0017.md

```diff
@@ -3,7 +3,30 @@ id: ErrorsCore.W0017
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Occurs a component's **width** or **height** properties have a value of the `function` type.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+If you need to specify a component's height or width and make the component's size responsive, use viewport units instead of a function. The most simple replacement is the following:
+
+    <!--JavaScript-->
+    // A value of the deprecated function type 
+    width: function() {
+        return window.innerWidth / 1.5;
```

Comment:
I suggest to use:
 return window.innerWidth / 2;
instead of 
 return window.innerWidth / 1.5;
Because it's more oblivious relation of width: 50vw instead of 66vw